### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by Junyi00**
## Changes
* Update language used in quick start
* Add more to FAQ and separate them by sections
* Fix command summary table has an extra column
* Update my PPP for changes here

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -26,13 +26,13 @@ Are you excited yet? Let's get started!
 
 ## Quick start
 
-1. Ensure you have Java `11` or above installed in your computer.
-   * If you have not installed Java, you can download and install Java for your computer from this [official website](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html).
-   * If you already have Java installed, you can refer to this [instruction](https://appuals.com/check-java-version-win-mac/) to verify the version of Java installed.
+1. Ensure your computer has Java `11` or above installed.
+   * If you need to install Java, the [official website](https://www.oracle.com/sg/java/technologies/javase/jdk11-archive-downloads.html) provide download links for most computer systems.
```
Comment (CloudHill) [inline docs/UserGuide.md]: Tiny error: should be "provides" instead of "provide"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1139,23 +1139,39 @@ Customer and order data are saved in the hard disk automatically after any comma
 
 ## FAQ
 
+### Customers
+{: .no_toc}
```
Comment (CloudHill) [inline docs/UserGuide.md:1143]: Is it intended that only the Customers section is not included in the toc?

**Review by CloudHill — CHANGES_REQUESTED** [review]
Very tiny nit

File: docs/UserGuide.md (hunk 2 above)
Comment (Junyi00) [reply docs/UserGuide.md:1143]: Thanks for pointing it out!

**Review by CloudHill — APPROVED** [review]
LGTM!

**Pull request merged**